docs/ensembl-help/getting-started/sequences.md

```diff
@@ -15,8 +15,16 @@ status: draft
 
 You can export sequences from the Genome Browser and Entity viewer apps using the Sequence download tool:
 
+## Genome browser
+
 ![](../../img/id-genome-browser.svg)
+In the genome browser, you can download gene and transcript sequences by clicking on the quick download link in the drawer or by clicking on the interactive transcript model and selecting the appropriate sequence to download.
+
+You can either download as FASTA, suitable for using with sequence analysis tools, or as rich text format (RTF), for visual analysis.
```

Comment:
Sequences can only be downloaded in the FASTA format via the quick download tool. At least that's how it is at the moment.